charts/chronograf/README.md

```diff
@@ -11,48 +16,38 @@ helm repo add influxdata https://helm.influxdata.com/
 helm upgrade --install chronograf influxdata/chronograf --namespace monitoring
```

Comment:
Do ppl need to install InfluxData Helm repository before installing the Helm chart?

Reply:
Yes, the repository must be added to Helm before they can install one of our charts